### PR TITLE
Adds New command

### PR DESCRIPTION
**Pull request by SweeXordious**
Now we have two commands for `OpenAPI`:
- `New` :  which generates a whole project with its gradle files
- `Generate` : only generates the `Core` interfaces and their `Implementations`

### CODE REVIEW

**Comment by xaviarias** [comment]
Could you please also remove the version provider from the `OpenApiCommand` class?

It is not used anymore as a subcommand of Epirus. Thanks !

**Comment by SweeXordious** [comment]
@xaviarias That `version provider` is used to have the same version of the `codegen` generating the project and the artifacts (server, core) in a generated project. Instead, of having to change it every time we release a new version.

**Comment by xaviarias** [comment]
Is very strange that `epirus --version` and `epirus openapi --version` don't have the same result.

How the user knows what that even means?

I think we should remove it and have only one version for the Epirus command.

**Comment by SweeXordious** [comment]
@xaviarias and how to have the same version for the `artifacts` and the `codegen` ? just hard code it  ?

**Comment by xaviarias** [comment]
Epirus has its own version provider.

I would just leave that as the only version.

**Comment by SweeXordious** [comment]
Cool. I will make the changes

**Comment by SweeXordious** [comment]
@xaviarias What do you think now ? This won't work until we merge the `Hard codes version` PR

File: src/main/kotlin/io.epirus.console/openapi/OpenApiCommand.kt [hunk 1]
```diff
@@ -12,24 +12,19 @@
...
-import java.time.format.DateTimeFormatter
-import java.util.Properties
 import java.util.concurrent.Callable
 
 @Command(
     name = "openapi",
     description = ["Generate an OpenAPI project"],
-    versionProvider = OpenApiCommand.VersionProvider::class,
-    subcommands = [GenerateCommand::class],
+    versionProvider = EpirusVersionProvider::class,
```
Comment (xaviarias) [inline src/main/kotlin/io.epirus.console/openapi/OpenApiCommand.kt:26]: This command shouldn't need a version, only Epirus will.

This should be removed.

Comment (SweeXordious) [reply src/main/kotlin/io.epirus.console/openapi/OpenApiCommand.kt]: done

**Comment by xaviarias** [comment]
> @xaviarias and how to have the same version for the `artifacts` and the `codegen` ? just hard code it ?

What is `artifacts`, is it a module?

**Pull request merged**